src/elexmodel/models/BaseElectionModel.py

```diff
@@ -111,6 +111,15 @@ def _get_reporting_aggregate_votes(self, reporting_units, unexpected_units, aggr
 
         return aggregate_votes
 
+    def _get_nonreporting_aggregate_votes(self, nonreporting_units, unexpected_units, aggregate, estimand):
```

Comment:
please add a unit test for this function

Reply:
Added!